README.md

```diff
@@ -1,6 +1,10 @@
 # 'OBE' (open-block-explorer)
 
-## Introduction
+Telos Main Net: [https://explorer.telos.net](https://explorer.telos.net)  [![Netlify Status](https://api.netlify.com/api/v1/badges/dc86b4ba-d574-4594-8f3c-88976edea863/deploy-status)](https://app.netlify.com/sites/open-block-explorer/deploys)
+
+Telos Test Net: [https://explorer-test.telos.net](https://explorer-test.telos.net) [![Netlify Status](https://api.netlify.com/api/v1/badges/8a778d87-a057-472f-ab3b-07d978faafaa/deploy-status)](https://app.netlify.com/sites/open-block-explorer-dev/deploys)  
+
```

Comment:
Just wondering if we should add the link to the development branch as well

Reply:
at the moment, there isn't a 'staging' site for OBE.